source/tools/QueueCache.h

```diff
@@ -0,0 +1,160 @@
+/**
+ *  @note This file is part of Empirical, https://github.com/devosoft/Empirical
+ *  @copyright Copyright (C) Michigan State University, MIT Software license; see doc/LICENSE.md
+ *  @date 2016-2020
+ *
+ *  @file  QueueCache.h
+ *  @brief
```

Comment:
Delete node, add Brief. Set date to 2020

Reply:
Done in #053f916.